doc/examples/applications/plot_tool_using_holoviz.py

```diff
@@ -0,0 +1,202 @@
+"""
+==========================================================
+Explore and annotate image data interactively with HoloViz
+==========================================================
+
+In this tutorial, we build interactive tools and an application using
+`Panel <https://panel.holoviz.org>`_ and `HoloViews <https://holoviews.org>`_.

```

Comment:
```suggestion
In this tutorial, we build interactive tools using
`Panel <https://panel.holoviz.org>`_ and `HoloViews <https://holoviews.org>`_,
two visualization libraries maintained by `HoloViz <https://holoviz.org/>`_. 
```
Since 'HoloViz' appears in the title, it must appear in the body text at some point, preferably towards the beginning, so the reader knows what we are talking about...